confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java

```diff
@@ -265,9 +266,9 @@ TDataPartitionTableResp getOrCreateDataPartition(
   /**
    * Register ConfigNode when it is first startup
    *
-   * @return TSStatus
+   * @return TConfigNodeRegisterResp
    */
```

Comment:
From my perspective, I think if there are no explain of param, there are no need to mention in java doc.